src/intervals/rounding.jl

```diff
@@ -116,6 +116,23 @@ for T in (Float32, Float64)
 end
 
 
+
```

Comment:
Could you add inline comment to clarify what's going on in there ? (after some research in `RoundingEmulator.jl` doc I think I get it, but future us will probably be thankful to have a bit of inline doc here)